concent_api/concent_api/settings/base.py

```diff
@@ -331,3 +331,12 @@
 
 # Verifier setting defining number of threads used by Blender
 BLENDER_THREADS = 1
+
+# Defines IP or domain name that can be used to connect to Middleman.
+# MIDDLEMAN_ADDRESS = ''
+
+# The port on which Middleman is accepting requests.
+# MIDDLEMAN_PORT =
```

Comment:
These two settings should be added to `base.py` in previous commit

Reply:
:+1: done